internal/provider/harness_base_resource.go

```diff
@@ -155,25 +210,21 @@ func (r *HarnessResource) ShouldSkip(ctx context.Context, req resource.CreateReq
 		return false
 	}
 
-	skip := false
 	for _, feat := range feats {
 		for pk, pv := range r.store.labels {
 			fv, ok := feat.Labels[pk]
 			if ok && (fv != pv) {
 				// if the feature label exists but the value doesn't match, skip
-				skip = true
-				break
+				return true
```

Comment:
Wondering if it could still be a good idea to have a warning diagnostic be written here.

Reply:
another good callout! I changed things to plumb through a `diag.Diagnostics` instead of an `error`, so these (and future) warnings can be surfaced in a more appropriate terraform way 🙏